config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, only: %i[new create show]
+  get 'sessions/new', to: 'sessions#new' 
+  post 'sessions', to: 'sessions#create'
+  delete 'sessions', to: 'sessions#destroy'
```

Comment:
# OPTIONAL CHANGES ♻️ 
- [ ] Refactoring this code to have at least one root route will be an awesome User Experience.

Reply:
A root route has now been created, directing users to events#index